.github/workflows/release.yml

```diff
@@ -265,7 +265,93 @@ jobs:
         retention-days: 5
 
   linux:
-    name: Linux
+    name: Linux (Generic)
+    needs: source
+
+    runs-on: ubuntu-20.04
+    container:
+      # manylinux2014 is based on CentOS 7, but already has a lot of things
+      # installed and preconfigured. It makes it easier to build OpenTTD.
+      image: quay.io/pypa/manylinux2014_x86_64
+
+    steps:
+    - name: Download source
+      uses: actions/download-artifact@v2
+      with:
+        name: internal-source
+
+    - name: Unpack source
+      run: |
+        tar -xf source.tar.gz --strip-components=1
+
+    - name: Install dependencies
+      run: |
+        echo "::group::Install dependencies"
+        yum install -y \
+          fontconfig-devel \
+          freetype-devel \
+          libicu-devel \
+          libpng-devel \
+          libpng-devel \
+          lzo-devel \
+          SDL2-devel \
+          wget \
+          xz-devel \
+          zlib-devel \
+          # EOF
+        echo "::endgroup::"
+
+        # The yum variant of fluidsynth depends on all possible audio drivers.
+        # This is not really useful for us, as that would require a user to
+        # have them all before he can start OpenTTD. Instead, compile a
+        # version that can only use SDL2. As OpenTTD does sound via SDL2,
+        # this simply means we either have sound and music, or have none.
```

Comment:
This is incorrect. OpenTTD does not depend on fluidsynth having output via a sound driver at all. The fluidsynth music driver sets up fluidsynth to call an internal callback to deliver samples, which are then mixed by our own sound mixer. If it's possible to build fluidsynth without any output drivers at all, you may as well do that.

https://github.com/OpenTTD/OpenTTD/blob/a667ed945f928a605a7e4c636d62d47eeb8dc6aa/src/music/fluidsynth.cpp#L68-L71